server/svix-server/src/cfg.rs

```diff
@@ -105,6 +105,28 @@ pub struct ConfigurationInner {
     pub api_enabled: bool,
     /// Should this instance run the message worker
     pub worker_enabled: bool,
+
+    #[serde(default = "Default::default", flatten)]
```

Comment:
Why is this needed?

Reply:
It was so the structure doesn't have to appear in the default config file, but I'll check how that works with flatten